integration/linux/env/vars.sh

```diff
@@ -140,8 +140,10 @@ TBBROOT=$(get_script_path "${vars_script_name:-}")/..
 
 TBB_TARGET_ARCH="intel64"
 
-if [ ! -z "$SETVARS_ARGS" ] && [ "$SETVARS_CALL" "==" "1" ]; then
-    for arg in "$SETVARS_ARGS"; do
+# shellcheck disable=2236
```

Comment:
Why do you prefer to suppress it but not fix?
Explanations in https://github.com/koalaman/shellcheck/wiki/SC2236 look reasonable.